layouts/shortcodes/location.html

```diff
@@ -0,0 +1,8 @@
+{{ $location := index .Site.Data.locations (.Get 0) }}
+
+<h2>Location</h2>
+<p>The tech day will be held in <a href="{{ $location.link }}">{{ $location.name }}</a>{{ if $location.detail }} at {{ $location.detail }}{{ end }}</p>
+
+<blockquote>
```

Comment:
I realise this was a blockquote previously, though I wonder whether there's a better option now that we're templating full HTML (rather than just markdown)? If there isn't something obvious/easy, this is fine.

Reply:
I quite liked how that looked. Perhaps in future we can embed something like a map, but I feel for now this is fine.